docs/data-processing.md

```diff
@@ -90,25 +90,83 @@ Continuing with the AMS-III.D. example, we map all twenty-two "known strings" th
+- carbon-capture: offsets derived from technologies that capture and store carbon
+- land-use: offsets derived from land management changes outside of forests
+- biochar: offsets derived from biochar production and application
+
+Category assignment is primarily determined by project type through the mapping defined in [`offsets-db-data/configs/type-category-mapping.json`](https://github.com/carbonplan/offsets-db-data/blob/main/offsets_db_data/configs/type-category-mapping.json). This mapping connects specific project types (like "improved forest management" or "cookstoves") to their appropriate category.
+
+### Project type assignment
+
+Project types represent more specific offset approaches. For example, within "forest" category, projects might be classified as "improved forest management", "afforestation/reforestation", or "avoided forest conversion".
+
+Project types are determined through a multi-step process:
+
+1. first, we attempt to infer the project type from protocol information (via {py:obj}`offsets_db_data.projects.infer_project_type`)
+2. then we apply manual overrides from curated data sources (via {py:obj}`offsets_db_data.projects.override_project_types`)
+3. the [Berkeley Carbon Trading Project](https://gspp.berkeley.edu/research-and-impact/centers/cepp/projects/berkeley-carbon-trading-project) data in [`offsets-db-data/configs/berkeley-project-types.json`](https://github.com/carbonplan/offsets-db-data/blob/main/offsets_db_data/configs/berkeley-project-types.json) serves as an authoriative source for project types.
+
+### Challenges in categorization
 
 The borders between these categories often blur.
 That is especially the case with energy efficiency and fuel switching protocols.
 Many protocols in these categories allow for projects that accomplish some mixture of the two approaches for displacing and reducing greenhouse gas emissions.
 Despite this blurriness, we assign each protocol to a single category.
-In the future, we are especially interested in adding project-level categorizations to the data.
-These project categories would highlight various sub-types of projects.
-This, for example, would provide an approach for distinguishing hydroelectric projects from wind projects, both of which fall within the renewable energy category.
 
-Finally, several smaller categories exist within the data (e.g., biochar).
+## Retirement beneficiary harmonization
+
+Carbon offset credits are often retired on behalf of a specific entity or organization. However, the names of these beneficiaries appear inconsistently across registry data, making it difficult to analyze retirement patterns.
+
+### Harmonization process
+
+We try to standardize retirement beneficiary information across registries using the following steps:
+
+1. **Data merging**: we combine information from four sources into a single `merged_beneficiary` field:
```

Comment:
I'm a bit confused. How does `merged_beneficiary` relate to `retirement_beneficiary_harmonized`?

Reply:
`merged_beneficiary` is a temporary field. we then run the transformations on this field to produce the final `retirement_beneficiary_harmonized`

Reply:
Gotcha -- in that case do you think it's necessary to introduce in the docs?

Reply:
just removed it